src/app/features/home/components/advertisement/rent-cards/rent-cards.component.ts

```diff
@@ -1,21 +1,20 @@
 import { Component, OnInit } from '@angular/core';
-
 import { Observable } from 'rxjs';
 import { CardService } from '../../../services/card.service';
 import { ICard } from '../../../../../core/models/cards';
 
 @Component({
   selector: 'app-rent-cards',
   templateUrl: './rent-cards.component.html',
-  styleUrls: ['./rent-cards.component.scss']
+  styleUrls: ['./rent-cards.component.scss'],
 })
 export class RentCardsComponent implements OnInit {
 
-  cards: Observable<Array<ICard>>|undefined;
+  cards: Observable<Array<ICard>> | undefined;
```

Comment:
We have fixed tsconfig file previously. You don't need | undefined or ?. Please fix across the app